docs/types.rst

```diff
@@ -96,6 +96,10 @@ In Solidity, division rounds towards zero. This mean that ``int256(-5) / int256(
 Note that in contrast, division on :ref:`literals<rational_literals>` results in fractional values
 of arbitrary precision.
 
+The modulus operator ``%`` yields the remainder from the division of the left operand by the right operand.
+Modulus results in the same sign as its left operand (or zero), for example ``int256(5) % int256(2) == int256(1)``
+and ``int256(-5) % int256(2) == int256(-1)``, respectively.
```

Comment:
Please explain how negative second operand behaves. Also, does it hold that `x % y == -((abs(x) % y)` for negative x? IF yes, please mention.